packages/react/src/stories/ic-text-field.stories.mdx

```diff
@@ -471,12 +447,18 @@ export const Uncontrolled = () => {
     parameters={{ loki: { skip: true } }}
   >
     <IcTextField
-      max-characters={2}
+      maxCharacters={2}
```

Comment:
I'm not sure if the warning behaviour is correct - it's displaying when i'm at the limit of characters, i assumed it'd only happen when i'm *over* the limit
<img width="360" alt="Screenshot 2024-07-05 at 15 24 50" src="https://github.com/mi6/ic-ui-kit/assets/111292477/7581832c-4718-4648-9e7b-75796d86d260">